Core1Platform/02_CoreCapabilities.md

```diff
@@ -0,0 +1,51 @@
+# Core capabilities
```

Comment:
I would suggest a different structure.

First emphasize the use cases. That it can run standalone or be integrated within an existing/new IT-infrastructure.

After that name the business cases it can handle

Reply:
Wenn erledigt, resolve conversation.